fast64_internal/oot/oot_constants.py

```diff
@@ -1494,3 +1070,183 @@
 	('12', 'To Dim Unk', 'Fade gSaveContext.unk_1410 255>100, works with scene xn 11/17'),
 	('13', 'From Dim', 'Alpha 100>255')
 ]
+
+ootEnWonderItemDrop = [
```

Comment:
Is there not some way to make all of this be handled within the XML rather than hardcoded within the plugin itself?

Reply:
Yeah that's a good idea, I'll move the enums I added to the XML